src/main/java/by/exadel/internship/controller/AuthController.java

```diff
@@ -29,10 +30,15 @@
     @ApiOperation("Authorize method")
     public ResponseEntity<?> authUser(@RequestBody LoginRequest loginRequest) {
         splitEmail(loginRequest);
-        Authentication authentication = authenticationManager
-                .authenticate(new UsernamePasswordAuthenticationToken(
-                        loginRequest.getLogin(),
-                        loginRequest.getPassword()));
+        Authentication authentication;
+        try {
+             authentication = authenticationManager
+                    .authenticate(new UsernamePasswordAuthenticationToken(
+                            loginRequest.getLogin(),
+                            loginRequest.getPassword()));
+        }catch (Exception e){
```

Comment:
What exception is actually thrown here? Isn't it BadCredentialsException by default?

Reply:
Actually we got NotFoundException, if we can't find user by login.

Reply:
So it should some more specific exception I believe

Reply:
Done. That was hard xD